src/services/codefixes/convertToAsyncFunction.ts

```diff
@@ -448,7 +442,7 @@ namespace ts.codefix {
         return createNodeArray([]);
     }
 
-    function getLastCallSignature(type: Type, checker: TypeChecker): Signature {
+    function getLastCallSignature(type: Type, checker: TypeChecker): Signature | undefined {

```

Comment:
The reason this compiled before is that `type` is declared non-optional. But it's tested for. Either the test is unnecessary, or `type` should be optional and some argument to this function is typed as non-optional when it shouldn't be.

Reply:
@andy-ms If `callSignatures` exists, is it guaranteed to have non-zero length? That was more of my motivation for the return value possibly being `undefined`.

Reply:
Oh, right. I wouldn't count on it. There is a `lastOrUndefined` function which has the correct type for potentially empty arrays.

Reply:
Changed to use `lastOrUndefined`.